hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2495,7 +2495,7 @@ func reconcileControlPlaneOperatorDeployment(
 
 	if hc.Spec.AdditionalTrustBundle != nil {
```

Comment:
this will be confusing to read with no context, can we also change the if statement 

```suggestion
	if hcp.Spec.AdditionalTrustBundle != nil {
```

Reply:
added suggestion and amended latest commit.